R/broom_wrapper.R

```diff
@@ -520,7 +525,13 @@ prediction_binary <- function(df, threshold = 0.5, ...){
       NULL
     }
   } else {
-    NULL
+    if(!is.null(df[["model"]][[1]]) && !is.null(df[["model"]][[1]]$y_levels)){
+      # this is new data prediction for xgboost_binary
+      # to check levels of response column
```

Comment:
Was not able to understand what "check levels of response column".
Please add some more explanation.